modules/kubernetes/fluxcd-v2-azdo/templates/azdo-proxy-values.yaml.tpl

```diff
@@ -1,6 +1,5 @@
-image:
-  tag: v0.3.6
```

Comment:
Why not define image.tag in the values file any more?

Reply:
This was wrong from the beginning and should be picked up by the application/chart version instead.